readme.md

```diff
@@ -227,7 +227,35 @@ bot.startRTM(function(err,bot,payload) {
   if (err) {
     throw new Error('Could not connect to Slack');
   }
+
+  // close the RTM for the sake of it in 5 seconds
```

Comment:
This is in the `readme.md`, you want JSDoc style comments in the README? Also *I DID* read CONTRIBUTING.md before submitting and it doesn't mention JSDoc at all.